src/common/dao/dao_test.go

```diff
@@ -871,27 +834,27 @@ func TestAddProjectMember(t *testing.T) {
 func TestUpdateProjectMember(t *testing.T) {
 	err := UpdateProjectMember(currentProject.ProjectID, 1, models.GUEST)
 	if err != nil {
-		t.Errorf("Error occurred in UpdateProjectMember: %v", err)
+		t.Fatalf("Error occurred in UpdateProjectMember: %v", err)
 	}
 	roles, err := GetUserProjectRoles(1, currentProject.ProjectID)
 	if err != nil {
-		t.Errorf("Error occurred in GetUserProjectRoles: %v", err)
+		t.Fatalf("Error occurred in GetUserProjectRoles: %v", err)
 	}
-	if roles[0].Name != "guest" {
-		t.Errorf("The user with ID 1 is not guest role after update, the acutal role: %s", roles[0].Name)
+	if assert.Len(t, roles, 1) {
+		assert.True(t, roles[0].Name != "guest",
```

Comment:
Should be roles[0].Name == "guest". I think using assert.Equal() here is better.